package/recalbox-system/xu4/recalbox.conf

```diff
@@ -104,6 +104,7 @@ controllers.bluetooth.enabled=1
 ##Enable PS3 controllers support
 controllers.ps3.enabled=1
 ## Choose an driver beetween official, shanwan and gasia if you have dualshock clones (official,shanwan,gasia)
+## or the new bluez (oficial, shanwan)
```

Comment:
No idea of how the shanwan driver works with genuine and gasia. May work the same as the bluez driver